### ISSUE TITLE
Config.toml error with Hugo-0.18

### ISSUE THREAD

**Issue by mdhender** [digitalcraftsman/hugo-steam-theme#13]
Getting an error with for hugo-0.18 for this theme:

```
ERROR: 2016/11/26 13:49:16 template.go:142: Unable to render [partials/themes/%!s(<nil>)-theme theme/partials/themes/%!s(<nil>)-theme]
ERROR: 2016/11/26 13:49:16 template.go:143: Expecting to find a template in either the theme/layouts or /layouts in one of the following relative locations [partials/themes/%!s(<nil>)-theme theme/partials/themes/%!s(<nil>)-theme]
```

This is the same error message as with Issue #3 but with a slightly different solution (the solution there was to include themeColor in the configuration file).

I was able to get around it by changing ```themeColor``` to ```themecolor``` in the ```partials/head.html``` file:

```
{{ partial (printf "themes/%s-theme" .Site.Params.themecolor) . }}
```

I didn't change the case of themeColor in the ```config.toml``` file. Is hugo-0.18 forcing everything to lowercase?

**Comment by digitalcraftsman** [comment]
Thanks for the report @mhender. Tomorrow, I'll investigate this issue further. But case-sensitive variables should have been fixed with spf13/hugo#2630.

**Comment by digitalcraftsman** [comment]
I added a workaround for this problem in 07c344f. But this does't solve the problem. Because I'm using camelCase all over the place some things still don't work:

- customizable strings like "Read more" aren't shown
- custom assets can't be added via the `customCSS` config variable

**Comment by digitalcraftsman** [comment]
I opened a new issue in the Hugo repo - see spf13/hugo#2740

**Comment by digitalcraftsman** [comment]
@mdhender did you used the latest commit to compile v0.18? Lowercasing `themeColor` in the template in combination with the latest version of Hugo it worked fine.

**Comment by digitalcraftsman** [comment]
Should be fixed in spf13/hugo#2741.

**Comment by mdhender** [comment]
Just did a new pull from spf14/hugo master and confirmed that `themeColor` is working as expected. Thanks.
